### PR TITLE
Try apidoc build in ReadTheDocs pipeline

### PR DESCRIPTION
**Pull request by KPLauritzen**
Force apidoc build in readthedocs builder

### CODE REVIEW

**Comment by KPLauritzen** [comment]
See https://dfds-ds-toolbox--5.org.readthedocs.build/en/5/index.html for PR build of the docs

File: docs/conf.py [hunk 1]
```diff
@@ -68,3 +68,19 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
+
+
+def run_apidoc(_):
```
Comment (dingobar) [inline docs/conf.py:73]: Do you have to copy paste the url or can it be posted here automatically somehow?

**Comment by dingobar** [comment]
> See https://dfds-ds-toolbox--5.org.readthedocs.build/en/5/index.html for PR build of the docs

Do you have to copy paste the url or can it be posted here automatically somehow?

**Comment by KPLauritzen** [comment]
> > See https://dfds-ds-toolbox--5.org.readthedocs.build/en/5/index.html for PR build of the docs
> 
> Do you have to copy paste the url or can it be posted here automatically somehow?

You can see it in readthedocs automated check
image

**Review by dingobar — APPROVED** [review]

**Pull request merged**